src/test/java/org/openmrs/contrib/qaframework/automation/RegistrationSteps.java

```diff
@@ -12,21 +11,23 @@
 import org.openmrs.reference.page.RegistrationPage;
 import org.openqa.selenium.By;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import io.cucumber.java.After;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
 
 public class RegistrationSteps extends Steps {
 
-	RegistrationPage registrationPage;
-	TestPatient patient;
-	ClinicianFacingPatientDashboardPage dashboardPage;
+	private RegistrationPage registrationPage;
+	private TestPatient patient;
+	private ClinicianFacingPatientDashboardPage dashboardPage;
 
-	@After(RunTest.HOOK.SELENIUM)
+	@After(RunTest.HOOK.SELENIUM_REGISTRATION)
 	public void destroy() {
 		quit();
 	}
 
-	@When("Registration user rightly logs in")
+	@Given("Registration user rightly logs in")
```

Comment:
Should n't the the user story read as` Registered user rightly logs in` ?

Reply:
> Should n't the the user story read as` @Given("Registered user rightly logs in")` ?

Sure the user story is read as Given as seen, Any mistake with that

Reply:
What do you say about `Registration user` vs `Registered user` ?

Reply:
> Registered user

Registered User sounds  good however it's different from registration user , ideally i think its the person who registers patient(registration clerk, registration nurse, registration doctor), not the already registered user(registered User. registered clerk registered nurse)

Reply:
According to the test implementation, the user story is talking of  login in of a registered user within the system!

Reply:
> registered user

Its talking of registration user, the person  who registers a patient forexample nurse, doctor , these person will become a user after him/she logs into the system

Reply:
To be little more detailed: This functionality should be the one in before method. so it should bring out the aspect of registration logs in not user logs in